src/bthread/task_control.cpp

```diff
@@ -42,6 +42,9 @@ DEFINE_int32(task_group_runqueue_capacity, 4096,
 DEFINE_int32(task_group_ntags, 1, "TaskGroup will be grouped by number ntags");
 DEFINE_bool(task_group_set_worker_name, true,
             "Whether to set the name of the worker thread");
+DEFINE_bool(thread_affinity, false, "Whether to Bind Cores");
+DEFINE_string(cpu_set , "",
```

Comment:
Maybe you can combine these two flags into one flag. If cpu_set is an empty string, then the thread affinity feature is off.